public/index.html

```diff
@@ -6,8 +6,169 @@
+                <option value="26">26</option>
+                <option value="27">27</option>
+                <option value="28">28</option>
+                <option value="29">29</option>
+                <option value="30">30</option>
+                <option value="31">31</option>
+            </select>
+
+            <select name="Year">
+                <option value="year">Year</option>
+                <option value="1980">1980</option>
+                <option value="1981">1981</option>
+                <option value="1982">1982</option>
+                <option value="1983">1983</option>
+                <option value="1984">1984</option>
+                <option value="1985">1985</option>
+                <option value="1986">1986</option>
+                <option value="1987">1987</option>
+                <option value="1988">1988</option>
+                <option value="1989">1989</option>
+                <option value="1990">1990</option>
+                <option value="1991">1991</option>
+                <option value="1992">1992</option>
+                <option value="1993">1993</option>
+                <option value="1994">1994</option>
+                <option value="1995">1995</option>
+                <option value="1996">1996</option>
+                <option value="1997">1997</option>
+                <option value="1998">1998</option>
+                <option value="1999">1999</option>
+                <option value="2000">2001</option>
+                <option value="2001">2002</option>
+                <option value="2002">2003</option>
+                <option value="2003">2004</option>
+                <option value="2004">2005</option>
+                <option value="2005">2006</option>
+                <option value="2006">2007</option>
+                <option value="2007">2008</option>
+                <option value="2008">2009</option>
+                <option value="2009">2010</option>
+                <option value="2010">2011</option>
+                <option value="2011">2012</option>
+                <option value="2012">2013</option>
+                <option value="2013">2014</option>
+                <option value="2014">2015</option>
+                <option value="2015">2000</option>
+            </select>
+           
+            <h3> What type of pets do you prefer? </h3>
+           
+            <select name="pet">
+                <option value="Pet">Pets</option>
+                <option value="dog">Dog</option>
+                <option value="cat">Cat</option>
+                <option value="bird">Bird</option>
+                <option value="bunny">Bunny</option>
+                <option value="ferret">Ferret</option>
+                <option value="lizard">Lizard</option>
+                <option value="other">Other</option>
+            </select>
+            
+            <div id="gender">
+                <input type="radio" name="sex" value="female"> Female
```

Comment:
Close. We didn't cover `label`s (realizing I totally missed them), but you should use a label here. I'm going to suggest doing some research into how to use labels for radio inputs and fixing this one.